pages/dashboard/views/index.jsx

```diff
@@ -9,6 +9,7 @@ const Index = ({ profile }) => (
     <Header title={<Snippet name={profile.firstName}>pages.dashboard.greeting</Snippet>} />
     <DashboardNavigation tab={0} />
     <h2><Snippet>pages.dashboard.tasks</Snippet></h2>
+    <Link page="search" searchType="tasks" label={<Snippet>tasklist.searchTasks</Snippet>} query={{ filters: { progress: ['open'] } }} />
```

Comment:
I'm assuming we want to remove this for the initial release and just browse directly to `/search/tasks` to test it?